ReportPortal.Tests/ReportPortal.E2E.API.Tests/Scenarios/Nunit/LaunchesQueryByUserNunitTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using ReportPortal.E2E.API.Core.Business;
+using ReportPortal.E2E.API.Core.Business.Models.Responses;
+using ReportPortal.E2E.API.Tests.TestData;
+using ReportPortal.E2E.Common.Extensions;
+
+namespace ReportPortal.E2E.API.Tests.Scenarios.Nunit
+{
+    [Parallelizable(ParallelScope.Fixtures)]
+    public class LaunchesQueryByUserNunitTests : BaseTest
```

Comment:
Design and implement test data storing utility and place it in Core Layer.

Reply:
Created for both runners. Thank you for remind

Reply:
Fix for all test data and all tests

Reply:
Done. Sorry, didn't catch that it should be for all tests